api/src/config/mod.rs

```diff
@@ -211,6 +217,7 @@ pub struct Config {
     applications: Applications,
     containers: Option<ContainerConfig>,
     jira: Option<JiraConfig>,
+    frontend: Option<FrontendConfig>,
```

Comment:
I'm trying to avoid the option blocks in the recent past and try to use a more idiomatic approach to this structs.

```suggestion
    #[serde(default)]
    frontend: FrontendConfig,
```

Reply:
Done.